pkg/reportportal/launch.go

```diff
@@ -0,0 +1,72 @@
+package reportportal
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"os"
+)
+
+const (
+	importFileFieldName = "file"
+)
+
+type RPLaunchService service
+
+func (s *RPLaunchService) Import(ctx context.Context, projectName string, importFile string) (*RPLaunchResponse, error) {
+	file, err := os.Open(importFile)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	fi, err := file.Stat()
+	if err != nil {
+		return nil, err
+	}
+
+	body := new(bytes.Buffer)
+	writer := multipart.NewWriter(body)
+	part, err := writer.CreateFormFile(importFileFieldName, fi.Name())
```

Comment:
Instand of using the zip file name we should use the pipeline name in the metadata to upload the test results so that the launcher name will be the pipeline name and not the zip file name 
```suggestion
	part, err := writer.CreateFormFile(importFileFieldName, launchName)
```